leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/LeshanClient.java

```diff
@@ -214,4 +214,8 @@ public void removeObserver(LwM2mClientObserver observer) {
     public String getRegistrationId() {
         return engine.getRegistrationId();
     }
+    
+    public RegistrationEngine getRegistrationEngine() {
+        return engine;
```

Comment:
Do you really need to access to RegistrationEngine ? if not, I will go for an update() or forceUpdate() method.
Does it make sense to you ?

Probably off-topic:  currently we support only one server but when we want support more I suppose we need to add a registrationID parameter to this method.

Reply:
You mean update() or forceUpdate() on LeshanClient directly? That would be OK too. Regarding the method naming, maybe the name triggerUpdateRegistration() is clearer. Just "update()" is not as clear, and "forceUpdate()" is maybe too harsh.